cmd/api/main.go

```diff
@@ -72,18 +71,18 @@ func main() {
 		path.Join(dirPath, "database.db"),
 	)
 
-	db, err := database.Open(databaseURL, attribute.String("database", "main"))
```

Comment:
We should open the database after `restoreBackup` because if the backup does not contain a migration the migration should be run. Otherwise, the migration would be run on an empty database and backup would overwrite that.

Reply:
good catch 👍